src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -4347,3 +4349,25 @@ void ScreenBufferTests::ClearAlternateBuffer()
 
     VerifyText(siMain.GetTextBuffer());
 }
+
+void ScreenBufferTests::InitializeTabStopsInVTMode()
+{
+    // This is a test for microsoft/terminal#411. Refer to that issue for more
+    // context.
+
+    auto& g = ServiceLocator::LocateGlobals();
+    auto& gci = g.getConsoleInformation();
+
+    VERIFY_IS_FALSE(gci.GetActiveOutputBuffer().AreTabsSet());
+
+    // Enable VT mode before we construct the buffer. This emulates setting the
+    // VirtualTerminalLevel reg key before launching the console.
+    gci.SetVirtTermLevel(1);

```

Comment:
Careful! Does this reset for subsequent tests?

Reply:
Good catch